projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_grid.scss

```diff
@@ -45,6 +45,7 @@
 /// @prop {Map} edit-mode-color [igx-color: ('secondary', 500)] - The text color in edit mode.
 /// @prop {Map} edited-row-indicator [igx-color: ('grays', 400)] - The indicator's color of edited row.
 /// @prop {Map} cell-edited-value-color [igx-color: ('grays', 600)] - The color of cell edited value.
+/// @prop {Map} cell-disabled-color [igx-color: ('grays', 600)] - The text color of a disabled cell.
```

Comment:
The assignment below states `igx-color: ('grays', 400)`, not 600;

Reply:
This is a number I seem to not have updated.

Reply:
And the true nuance is in the middle, we use grays 500 for disabled.

Reply:
should be 500